Luke_Smith_Portfolio_Contact.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta name="viewport" content="width=device-width initial-scale=1">
+	<title>Luke Smith Portfolio Contact</title>
+    <link rel="stylesheet" href="Luke_Smith_Portfolio.css">
+</head>
+<body>
+	<div style="background-color:white;">
```

Comment:
Remove the style attribute from this tag.

Reply:
I'm not sure this div tag is even necessary

Reply:
Nevermind, this div is providing the background colour. Nevertheless, it is bad practice to have two body tags. Move this div into the main body tag, and remember to close it at the end of the document.

It is still bad practice to declare styles inline. Add a class to this div (e.g. "backgroundcolour") and add the following entry to the .css:
.backgroundcolour {
background-color: #FFFFFF;
}